{{cookiecutter.project_slug}}/.github/workflows/cruft-update.yaml

```diff
@@ -1,16 +1,18 @@
-name: Pre-commit auto-update
+name: Cruft update
```

Comment:
Please rename to weekly or something similar, as it may include other upkeep jobs